### ISSUE TITLE
RFM - phpstorm

### ISSUE THREAD

**Issue by Spunkie** [chocolatey/chocolatey-package-requests#104]
https://chocolatey.org/packages/phpstorm

The pkg currently has 3 maintainers. As far as I can tell 2 of them have abandoned chocolatey completely and deleted their pkg repos. The 3rd maintainer, Claud, still has [a repo](https://github.com/Claud/chocolatey-packages) but has not responded to [requests for updates](https://github.com/Claud/chocolatey-packages/issues/7) in a long time.

@mastacheata said they would be open to being the new maintainer. I would also be open to being a secondary maintainer if needed.

**Comment by mastacheata** [comment]
As requested on the referenced issue, this is my chocolatey.org profile:
https://chocolatey.org/profiles/mastacheata

**Comment by Spunkie** [comment]
@mastacheata I added you a few days ago, you have have gotten an email from chocolatey asking for confirmation. 
___
Also I have made a few commits for you repo and would like to submit PRs but since generated files are committed in master it could cause many unnecessary merge conflicts or out of date files at times. Would you mind making a dev branch that deletes all those generated files and adds them to the .gitignore? 

I can then do my PRs against that branch and you can merge that into master once you are ready to push to chocolatey.

**Comment by mastacheata** [comment]
@Spunkie Landed in my junk mail, which I don't check so regularly, but found it now.

I removed the autogenerated installer, uninstaller and the nuspec file and added corresponding entries to the gitignore file.

I've fiddled a bit. Left the master branch untouched and created a dev branch where I removed all autogenerated files and added them to .gitignore
Just checked your fork, you added *.nuspec to the gitignore, but that will cause more trouble because the template file is also a .nuspec file. (i.e. phpstorm.nuspec - generated, phpstorm_template.nuspec - template file that must be kept)

Also created seperate branches for phpstorm and rubymine as I'm not the maintainer for RubyMine, but use that pretty regularly as well.
